src/device/input/Keyboard.cpp

```diff
@@ -26,18 +26,18 @@
 
 uint8_t Keyboard::normal_tab[] = {
         0, 0, '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', 225, 39, '\b',
-        0, 'q', 'w', 'e', 'r', 't', 'z', 'u', 'i', 'o', 'p', 129, '+', '\n',
+        0, 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', 129, '+', '\n',
```

Comment:
Please do not change the keyboard layout. Feel free to implement multiple layouts and a switching mechanism (this is a planned feature, anyways).